src/shared-components/Card/ListItems/ListItemSetting.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+} from '@mui/material';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import { colors } from 'src/styles/colors';
+import { theme } from 'src/styles/theme';
+
+export const ListItemSetting = ({
+  title,
+  description,
+}: {
+  title: string;
+  description: string;
+}) => {
+  return (
+    <Box display='flex' flexDirection='column' gap={theme.spacing(2)}>
+      <List>
```

Comment:
Remove list element, only keep listItem. Don't forget to clean imports.

Reply:
Fixed